components/atoms/Buttons/Button/Button.js

```diff
@@ -1,14 +1,17 @@
-import React from 'react'
-
-const styles ={
-  Static: 'w-[276px] h-[70px] rounded-[10px] bg-[#FD7A55] flex items-center justify-center drop-shadow-[0_15px_25px_rgb(#FFB39D)]'
-}
+import React from "react";
 
+const styles = {
+  Static:
+    "w-[276px] h-[70px] rounded-[10px] bg-[#FD7A55] flex items-center justify-center drop-shadow-[0_15px_25px_rgba(255,179,157)]",
+};
 
-export default function Static({children, onClick, type}) {
+export default function Static({ children, onClick, type }) {
```

Comment:
```suggestion
export default function Button({ children, onClick, type }) {
```

название компоненты должно быть таким же как название файла